src/entries/popup/components/CommandK/useSearchableNFTs.ts

```diff
@@ -84,10 +86,17 @@ export const useSearchableNFTs = () => {
 
   const searchableNFTs = useMemo(() => {
     return nfts.map<NFTSearchItem>((nft) => ({
-      action: () =>
-        navigate(
-          ROUTES.NFT_DETAILS(nft.collection.collection_id || '', nft.id),
-        ),
+      action: () => {
+        const [chainName, contractAddress, tokenId] = nft.fullUniqueId.split(
+          '_',
+        ) as [ChainName, Address, string];
+        const chainId = chainNameToIdMapping[chainName];
+        if (!chainId) return;
+        return navigate(
+          ROUTES.NFT_DETAILS(`${contractAddress}_${chainId}`, tokenId),
+          { state: { nft } },
+        );
+      },
```

Comment:
lol yes, 
but since token details work like that I made it the same, this way we can search for nfts the users doesn't own

Reply:
Do you think there's any value in conditionally fetching the nft if the data doesn't exist? Presumably the data in the details screen is slower to populate now. Would be nice to support this, but use things like the images if we already have them.

Reply:
it shouldn't be slower is it? since we only fetch if we don't have the data, but currently we always have the data

I agree it's was not necessary and will try not to do stuff thinking we may need later like that in the future, I believe we should code what's needed when needed and overcomplicated here thinking of a future case

Reply:
"but currently we always have the data" <-- I was just checking that that was the case and asking your opinion on conditional fetching. Sounds like you've already thought through it.

"and will try not to do stuff thinking we may need later like that in the future" <-- I think this was a good idea. I appreciate the forethought. I was just asking for some clarification. This change wasn't properly documented.